src/tox/config/loader/ini/factor.py

```diff
@@ -45,7 +45,7 @@ def explode_factor(group: List[Tuple[str, bool]]) -> str:
 
 def expand_factors(value: str) -> Iterator[Tuple[Optional[Iterator[List[Tuple[str, bool]]]], str]]:
     for line in value.split("\n"):
-        match = re.match(r"^((?P<factor_expr>[\w{}.!,-]+):\s+)?(?P<content>.*?)$", line)
+        match = re.match(r"^(\s*(?P<factor_expr>[\w{}.!,-]+):\s+)?(?P<content>.*?)$", line)
```

Comment:
Why do we need this part?

Reply:
Yesterday, before I wrote tests, I setup a minimal `tox.ini`  to reproduce the problem, and I could swear I head a leading whitespace or even a linebreak for `line` (which does not seem feasible in retro as it gets split at `\n`), and the fix to get it parsed correctly was to update the regex.

As I deleted the example and both the tests and the reported problem with the `tox.ini` in `virtualenv` are already fixed without the regex update, I will revert it (later that night). :+1: